### DISCUSSION TITLE
What is an "External" project?

### DISCUSSION THREAD

**Discussion by JohnMajerle** [iTwin/itwinjs-core#96] [General] [answered]
When I create an iModel using the **_Registration Dashboard_** the newly created project is marked **_External_**.  See image, below.  Projects I create manually from CONNECT do not have this designation.  The one big difference I see is that I don't appear to have any control over management of the user roles for users associated with this project, which I believe means I can't try out various services such as Forms.

If that is the case, how do I create an iModel in a non-External project?

![image](https://user-images.githubusercontent.com/30668739/88400227-a628dc80-cd8d-11ea-9588-c082fe2259a8.png)

**Comment by jan-slegr** [comment]
> If that is the case, how do I create an iModel in a non-External project?

In [CONNECT Center](https://connect.bentley.com/), open the (not external) project and use _iModel Manager_. It's available in _iTwin Services_ list.

![iModelManager](https://user-images.githubusercontent.com/8292864/88475834-cbefe600-cf33-11ea-92da-6cec5e8592d1.jpg)

**Comment by williamkbentley** [comment, marked as answer]
As noted on the [Getting started registration dashboard](https://www.imodeljs.org/getting-started/registration-dashboard?tab=1) iModels created using the dashboard are administered by Bentley and provide you limited access to security policies and services. The intent is to provide third party developers a starter iModel to use during development of iModel.js applications, not a full featured CONNECT project.

It is marked as 'External' because this is a project outside of your control that you have been invited to. You must create a project from within CONNECT to have full control over the project. 

@LabyrinthTechnology is correct, if you have access to the iModel Manager iTwin service, you should be able to create iModels that way. This is typically done by an administrator.
